gulpfile.coffee

```diff
@@ -125,8 +125,8 @@ gulp.task 'clean', ->
 
 gulp.task 'mocha', ['coffee'], ->
   gulp.src files.test, ['coffee'], { read: true }
-    .pipe mocha {reporter : 'nyan'}
-    # .pipe exit()
+    .pipe mocha {reporter : 'spec'}
```

Comment:
'list' is also a good choice. I haven't tried 'spec' though (hope its better than nyan ^^)